VirtoCommerce.Storefront.Model/Catalog/Services/ICatalogSearchService.cs

```diff
@@ -21,5 +22,7 @@ public interface ICatalogService
         CatalogSearchResult SearchProducts(ProductSearchCriteria criteria);
 
         IPagedList<Category> SearchCategories(CategorySearchCriteria criteria);
+
+        Task LoadAssociations(ProductAssociation[] productAssociations);
```

Comment:
Why do we need this very specific method in the interface?
I think we need to load associations based on responseGroup in both GetProductsAsync and SearchProductsAsync

Reply:
We don't use response groups when we working with expApi and current realization load only association ids in product object to lead "lazy loading" pattern, so we need an additional method to call it outside and inside service.